### PR TITLE
tests: Uniform mocking/mock only at tested module

### PR DESCRIPTION
**Pull request by tyll**

### CODE REVIEW

File: tests/test_l3_interface.py [hunk 1]
```diff
@@ -23,20 +23,20 @@
 except ImportError:  # py2
     import mock
 
-import sys
+mock_modules = {}
```
Comment (EdDev) [inline tests/test_l3_interface.py]: You could create a dict in some common place and return it directly in line 37.
Not sure about the __import__ portion: Does it matter where it is called?

Comment (tyll) [reply tests/test_l3_interface.py]: Yes, I thought about this as well... The import is different in each file.

Comment (EdDev) [reply tests/test_l3_interface.py]: So can you then do something like:
`mock_modules(__import__("ansible_nmstate"))` ?

Comment (tyll) [reply tests/test_l3_interface.py]: ah, I meant the import that is patched... this one could be part of the common code.

**Comment by EdDev** [comment]
@tyll , do you intend to locate this in a common place or leave it like this?

**Comment by tyll** [comment]
changed it to use a common file

**Review by EdDev — APPROVED** [review]
Maybe you can call the new module with a more specific name, like mocklib, or common_mock.
Such names usually absorb non related parts of the project (anything that you cannot give it a clear context or grouping name).

**Comment by tyll** [comment]
renamed it to testlib

**Pull request merged**